specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+                        "description": "Request accepted for deletion of attestation service"
+                    },
+                    "404": {
+                        "description": "Message not found"
+                    },
+                    "500": {
+                        "description": "Internal error",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Error result from Attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResult"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "OperationList": {
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of supported operations.",
+                    "items": {
+                        "$ref": "#/definitions/OperationsDefinition"
+                    }
+                }
+            },
+            "description": "List of supported operations."
+        },
+        "OperationsDefinition": {
+            "type": "object",
+            "description": "Definition object with the name and properties of an operation.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "Name of the operation."
+                },
+                "display": {
+                    "$ref": "#/definitions/OperationsDisplayDefinition",
+                    "description": "Display object with properties of the operation."
+                }
+            }
+        },
+        "OperationsDisplayDefinition": {
+            "type": "object",
+            "description": "Display object with properties of the operation.",
+            "properties": {
+                "provider": {
+                    "type": "string",
+                    "description": "Resource provider of the operation."
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "Resource for the operation."
+                },
+                "operation": {
+                    "type": "string",
+                    "description": "Short description of the operation."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of the operation."
+                }
+            }
+        },
+        "AttestationProvider": {
+            "type": "object",
+            "description": "Attestation service response message.",
+            "required": [
+                "properties"
+            ],
+            "x-ms-azure-resource": true,
+            "properties": {
+                "id": {
```

Comment:
For efficiency, it's best to use an allOf with pointer to the [common types ](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json) file, or to at least add a "resource" and/or "proxyResource" definition in your spec, rather than adding top-level properties to each resource type.

Reply:
add 
            "allOf": [
                {
                    "$ref": "#/definitions/Resource"
                }